### PR TITLE
Add information about environment variables to documentation

### PR DESCRIPTION
**Pull request by harrisj**
The main point of this pull request is to add some information about environment variables and the `Client#from_env_vars` constructor to the documentation. I also made a few other stylistic tweaks. Edits and feedback are always welcome. Do not merge into master until we decide this is no longer a WIP.

This pull request is in response to issue #3

### CODE REVIEW

**Comment by amoose** [comment]
pinging @alex and @llimllib for feedback.

**Comment by llimllib** [comment]
LGTM

File: docs/index.rst [hunk 1]
```diff
@@ -23,6 +22,32 @@ Once you have credentials for VBMS, you can dive in:
         '<path to client certificate, or nil>',
     )
 
+ALternatively, you can set the values for these fields as environment variables. This approach is preferred since it avoids checking in any credentials to a repository and is one of the requirements for a proper `12-factor application`_ deployment. The environment variables that you must define are:
```
Comment (alex) [inline docs/index.rst]: Typo "ALternatively"

**Comment by alex** [comment]
besides that typo LGTM

**Comment by harrisj** [comment]
Okay. I also realize I should add a note to the README.md about how to run the tests so they connect to the test server instead of the webmock. Also, can one of you help me better style the environment variables?

**Comment by llimllib** [comment]
How do you want them to look better? They look fine to me?

Also one thought that occurred to me is that maybe this change pushes the ToC too far down the index page of the documentation? Maybe the index.rst should be a brief message about what `connect_vbms` is, then right into the ToC. This would push the configuration information off into a new, separate page.

**Comment by llimllib** [comment]
(maybe that's outside the scope of this PR)

**Pull request merged**